server/src/main/kotlin/fi/oph/kitu/yki/entities/TutkintokieliEntity.kt

```diff
@@ -0,0 +1,14 @@
+package fi.oph.kitu.yki.entities
```

Comment:
Onko näin monimutkaselle kansiorakenteelle tarvetta täällä ykin alla? IMO nää enumit voisi olla ton ite suorituksen yhteydessä samassa filussa (ja sama ton responsen osalta). Tai sitten pitää muuttaa miten kotoutumiskoulutuksen alla ollaan tehty – mut en haluis et on tehty eri tavalla yki ja kotoutumiskoulutus.

Reply:
Mä laitoin nämä saman kansion alle, ettei menis niin helposti sekaisin, kun käytännössä pitää olla aina pitää olla `Response` - luokalle ja sille vastaava `Entity` - luokka. Mun mielestä näin taas löytyy helpommin halutut luokat.